http_server/queries/users/users_select_top.php

```diff
@@ -10,7 +10,7 @@ function users_select_top($pdo, $start, $count)
         FROM users, pr2, rank_tokens
         WHERE users.user_id = pr2.user_id
         AND pr2.user_id = rank_tokens.user_id
-        AND rank > 49
+        AND active_rank > 49
```

Comment:
I tried this same thing. It seems a computed value can't be used in this part of the query. :/

Reply:
@jacob-grahn could `HAVING` be used instead? https://forums.mysql.com/read.php?10,266846,267031#msg-267031